src/com/owncloud/android/lib/common/operations/RemoteOperationResult.java

```diff
@@ -119,7 +119,8 @@
         DELAYED_FOR_WIFI,
         LOCAL_FILE_NOT_FOUND,
         SERVICE_UNAVAILABLE,
-        SPECIFIC_SERVICE_UNAVAILABLE
+        SPECIFIC_SERVICE_UNAVAILABLE,
+        SPECIFIC_UNSUPPORTED_MEDIA_TYPE
```

Comment:
```just in case we want to ... in the future```

We need to be careful with things that might want or might now want in the future. Names are not something I would prepare "for the future", because they are easy to change if it's needed in the future, and because they can be misleading if no change is needed in the end.

```we are setting it when the response body contains a specific error message```

I would buy this if the name were ```UNSUPPORTED_MEDIA_TYPE_WITH_SPECIFIC_MESSAGE```, but the meaning of ```SPECIFIC_UNSUPPORTED_MEDIA_TYPE``` in plain English is not exactly the same.

Besides, if I'm not wrong we are setting this result if response is 415, no matter if comes with an specific message or not.